### PR TITLE
add Laravel Simple Password Manager

### PR DESCRIPTION
**Pull request by milenmk**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [ ] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [ ] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Review by Ki-er — CHANGES_REQUESTED** [review]
More changes to be made alongside the above.

**Comment by milenmk** [comment]
All proposed changes are made :)

**Review by kokomo123 — APPROVED** [review]
Thank you for committing the suggestions, i shall now approve this PR! 💯 ✅

File: software/password-manager.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: "Laravel Simple Password Manager"
+website_url: "https://github.com/milenmk/Laravel-Simple-Password-Manager"
+source_code_url: "https://github.com/milenmk/Laravel-Simple-Password-Manager"
+description: "Password manager written with PHP and Laravel.."
```
Comment (nodiscc) [inline software/password-manager.yml:4]:
```suggestion
description: "Simple Password manager."
```

File: software/password-manager.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+licenses:
+  - GPL-3.0
+platforms:
+  - PHP
+tags:
+  - Password Managers
+depends_3rdparty: true
```
Comment (nodiscc) [inline software/password-manager.yml:11]: What third-party service does it depend on?

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/password-manager.yml (hunk 2 above)
Comment (milenmk) [reply software/password-manager.yml]: I wasn't sure what to put here in the beginning. I've change it to false now. In reality, the software doesn't connect to any  API or website or any of this sort to work. On the other hand, it doesn't have a Docker image, so the prerequizite is to have LAMP stack or similar previously installed.

Comment (kokomo123) [reply software/password-manager.yml]: Feel free to omit it if it doesn't connect to third-party services, that line is if it depends on like Google, etc.

**Review by nodiscc — APPROVED** [review]

**Pull request merged**